package.json

```diff
@@ -79,6 +79,7 @@
     "clean-webpack-plugin": "^3.0.0",
     "commander": "^2.19.0",
     "css-loader": "^3.2.0",
+    "css-selector-tokenizer": "^0.7.2",
```

Comment:
These are two known bugs for this dependency: https://github.com/css-modules/css-selector-tokenizer/issues

Any worries that this could be a problem for our theming? I'm assuming no given our recommended/expected usage

Reply:
no, neither of those issues affect us